packages/cf-style-const/src/variables.js

```diff
@@ -0,0 +1,41 @@
+import cssColorFunction from 'css-color-function';
+
+const color = input => cssColorFunction.convert(`color(${input})`);
+
+const t = {};
```

Comment:
Because that way you can't self reference these values which we do a lot. We could be more strict and define constants at the top of file. Or we could even use currying. Not sure what annoys me less tho. :)